Update DCP_change_log.md (#2705)
SkylineCommunications/dataminer-docs: release-notes/Cloud/DCP_change_log.md

> [!NOTE]
> Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).

> #### 13 February 2024 - Fix - Catalog will show versions sorted in descending order [ID_38762]

Reviewer: ```suggestion
#### 13 February 2024 - Fix - Catalog versions displayed in wrong order [ID_38762]
```

> The version overview for a catalog record will show versions now sorted based on their version number instead of the order they were stored.

Reviewer: ```suggestion
The versions of a catalog will now be sorted correctly.
```
